packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Node.java

```diff
@@ -98,6 +100,14 @@ public INodeStyle getStyle() {
         return this.style;
     }
 
+    public ILayoutStrategy getChildrenLayoutStrategy() {
+        return this.childrenLayoutStrategy;
+    }
+
+    public static String getSelectedNode() {
+        return SELECTED_NODE;
+    }
```

Comment:
Does this come from another patch? It seems unrelated and unused by the rest of the PR.

Reply:
I will remove it